library/src/main/java/io/ona/kujaku/views/KujakuMapView.java

```diff
@@ -1211,6 +1213,21 @@ public void onMapReady(MapboxMap mapboxMap) {
         }
     }
 
+    @Override
+    public boolean changeLocationUpdates(long updateInterval, long fastestUpdateInterval, int accuracyLevel) {
```

Comment:
Instead of of `changeLocationUpdates`, how does `customizeLocationUpdates` sound?

Reply:
I personally think that `changeLocationUpdates` still sounds OK and feels more norm to me

Reply:
OK.